File: vox-study-core/src/main/scala/org/tyrant/study/socket/TimeServer.scala
package org.tyrant.study.socket

import java.io.PrintStream
import java.net.ServerSocket

/**
  * Created by cielleech on 2017/2/13.
  */
object TimeServer {
  def main(args: Array[String]): Unit = {
    val ss = new ServerSocket(8080)
    while(true) {
      val socket = ss.accept()
      val ps = new PrintStream(socket.getOutputStream)
      ps.println("Hello, Client!")
      ps.close()
      socket.close()
    }
  }
}

File: vox-study-core/src/main/java/org/vox/study/nio2/TimeServer.java
package org.vox.study.nio2;

/**
 * Created by cielleech on 2016/4/9.
 */
public class TimeServer {
    public    static void main(String[] args) {
        int port = 8080;
        if (args != null && args.length > 0) {
            try {
                port = Integer.valueOf(args[0]);
            } catch (RuntimeException e) {

            }

        }
    }
}
